### ISSUE TITLE
Mapping the Zapier

### ISSUE THREAD

**Issue by edumaciel10** [Open-Pix/zapier-woovi#4]
- [ ] Zapier need some request access?
- [ ] Does Zapier has some documentation step-by-step on how to develop?
- [ ] How do works the mindmap of integration with zapier

**Comment by igorfelipeduca** [comment]
Each request on Zapier needs to set request headers.
Every time an user uses our block in the Zap editor, he needs to pass his OpenPix API Key as input.

So, this way, I will authenticate the request to the API and perform the action.
```ts
const response = await z.request({
    method: "POST",
    url: `${process.env.BASE_URL}/charge`,
    body: {
      correlationID: bundle.inputData.correlationID,
      value: bundle.inputData.value,
    },
    headers: {
      "Authorization": bundle.inputData.API_KEY
    }
  });

return response.data;
```

**Comment by igorfelipeduca** [comment]
Yes, Zapier has a pretty great documentation, you can check it out [here](https://zapier.github.io/zapier-platform/).

**Comment by igorfelipeduca** [comment]
The mindmap of Zapier integrations works this way:

**Creates** -> You run a command at the Zap runtime, this command can send a POST to create something, like a charge, for example

**Trigger** -> The trigger is what keeps listening to OpenPix's webhooks. For example, if a charge is created, its request will trigger some specific trigger related to charges. Triggers are only listeners
